pom.xml

```diff
@@ -572,5 +572,97 @@
+              <additionalOptions>
+                <additionalOption>-hdf</additionalOption>
+                <additionalOption>book.path</additionalOption>
+                <additionalOption>/java/_book.yaml</additionalOption>
+                <additionalOption>-hdf</additionalOption>
+                <additionalOption>project.path</additionalOption>
+                <additionalOption>/_project.yaml</additionalOption>
+                <additionalOption>-hdf</additionalOption>
+                <additionalOption>devsite.path</additionalOption>
+                <additionalOption>/java/docs/reference/</additionalOption>
+                <additionalOption>-d</additionalOption>
+                <additionalOption>${project.build.directory}/apidocs</additionalOption>
+                <additionalOption>-templatedir</additionalOption>
+                <additionalOption>${devsite.template}</additionalOption>
+                <additionalOption>-toroot</additionalOption>
+                <additionalOption>/java/docs/reference/</additionalOption>
+                <additionalOption>-yaml</additionalOption>
+                <additionalOption>_toc.yaml</additionalOption>
+                <additionalOption>-warning</additionalOption>
+                <additionalOption>101</additionalOption>
+              </additionalOptions>
+              <useStandardDocletOptions>false</useStandardDocletOptions>
+              <additionalJOption>-J-Xmx1024m</additionalJOption>
+            </configuration>
+          </plugin>
+          <plugin>
+            <!-- Clean up some references and files. -->
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>1.7</version>
+            <executions>
+              <execution>
+                <phase>site</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                  <target>
+                    <echo message="Updating relative links in API docs" />
+                    <replace dir="${project.build.directory}/apidocs" token="href=&quot;com" value="href=&quot;/java/docs/reference/com" />
+                    <copy file="${project.build.directory}/apidocs/assets/_toc.yaml" todir="${project.build.directory}/apidocs/reference" />
+                    <delete file="${project.build.directory}/apidocs/reference/classes.html" />
+                    <delete file="${project.build.directory}/apidocs/reference/hierarchy.html" />
+                    <delete file="${project.build.directory}/apidocs/reference/index.html" />
+                    <delete file="${project.build.directory}/apidocs/reference/lists.js" />
+                    <delete file="${project.build.directory}/apidocs/reference/package-list" />
+                    <delete file="${project.build.directory}/apidocs/reference/packages.html" />
+                    <delete file="${project.build.directory}/apidocs/reference/current.xml" />
```

Comment:
Most of these files are generally pretty core to navigating java docs. Why are they being removed?

Reply:
If the new doclet is taking care of replacing the default frame navigation and full list views it's probably okay that these are being removed. 

Do you have a template I could use to build the site locally on my workstation to test out/poke around?